dev/tasks/tasks.yml

```
@@ -875,6 +875,15 @@ tasks:
       image: ubuntu-cpp
 {% endfor %}
 
+  test-skyhook-integration:
+    ci: github
+    template: docker-tests/github.linux.yml
+    params:
+      flags: -e ARROW_SKYHOOK=ON
+      env:
+        UBUNTU: 20.04
```

We're sorting this in alphabetical order.

```suggestion
      env:
        UBUNTU: 20.04
      flags: -e ARROW_SKYHOOK=ON
```